src/pages/community.js

```diff
@@ -1,71 +1,255 @@
+    margin-right: 18px;
+  }
+`
+
+const MeetupLink = styled(Link)`
+  font-size: 16px;
+`
+
+const TwitterWidgetsOnlyOnClientSide = () => {
+  if (typeof window !== 'undefined') {
+    // package react-twitter-embed does not work on SSR, therefore it breaks the Gatsby build
+    const {
+      TwitterTimelineEmbed,
+      TwitterFollowButton,
+    } = require('react-twitter-embed')
+
+    return (
+      <React.Fragment>
+        <TwitterFollowButton
+          screenName="reactjsacademy"
+          options={{ size: 'large' }}
+        />
+        <TwitterTimelineEmbed
+          sourceType="profile"
+          screenName="reactjsacademy"
+          options={{ height: 800 }}
+        />
+        <TwitterFollowButton
+          screenName="reactjsacademy"
+          options={{ size: 'large' }}
+        />
+      </React.Fragment>
+    )
+  } else {
+    return null
+  }
+}
+
 const Community = () => (
   <React.Fragment>
     <Header
       titleLines={['The ReactJS', 'Academy community']}
       subtitle="We are not a group of people - but a movement!"
+      links={[
+        { text: 'Twitter ', to: '#twitter' },
+        { text: 'Meetups', to: '#meetups' },
+        { text: 'Instagram', to: '#instagram' },
+        { text: 'Mentor community', to: '#mentor-community' },
+      ]}
       bgImg="training-event"
     />
     <TopSection>
       <Grid>
-        <Card border="shadow">
-          <Row>
-            <Col xs={12} lg={10} lgOffset={1}>
-              <H2>Trusted by industry leaders</H2>
-              <TrustedByLogoList />
-            </Col>
-          </Row>
-        </Card>
+        <Row>
+          <Col xs={12} md={6}>
+            <Card border="shadow">
+              <Col md={8} mdOffset={2}>
+                <H2>
+                  Twitter? Sure.
+                  <a name="twitter" />
+                </H2>
+                <CallToAction
+                  cta
+                  to={nextBootcamp && nextBootcamp.pathUrl}
+                  children={`Next Bootcamp: ${nextBootcampStartDate}, ${
+                    nextBootcamp.cityShortName
+                    } >>`}
+                />
+                <TwitterWidgetsOnlyOnClientSide />
+              </Col>
+            </Card>
+          </Col>
+          <Col xs={12} md={6}>
+            <Card border="shadow">
+              <Col md={8} mdOffset={2}>
+                <H2>
+                  Meetups? Absolutely! <a name="meetups" />
+                </H2>
+                {meetups.length ? (
+                  <React.Fragment>
+                    <EventList>
+                      <Li>
+                        <H3>Events</H3>
+                      </Li>
+                      {meetups.map(
+                        ({
+                          cityShortName,
+                          country,
+                          dateStartsOn,
+                          url,
+                          imgUrl,
+                        }) => (
+                            <Li>
```

Comment:
when you run `yarn test`, it throws this warning:

```
 PASS  src/pages/commuity.test.js
  ● Console

    console.error node_modules/fbjs/lib/warning.js:33
      Warning: Each child in an array or iterator should have a unique "key" prop.

      Check the top-level render call using <styled.ul>. See https://fb.me/react-warning-keys for more information.
          in styled.li
    console.error node_modules/fbjs/lib/warning.js:33
      Warning: Each child in an array or iterator should have a unique "key" prop.

      Check the top-level render call using <Row>. See https://fb.me/react-warning-keys for more information.
          in Styled(Col)
```
it needs a `key` prop on every child :)

Reply:
@horacioh fixed! thanks